### PR TITLE
style done for footer component

### LINKED ISSUE #3: [Footer] Implement and style footer
- [x] Desktop Version
Screenshot 2024-07-12 at 16 24 23
Screenshot 2024-07-12 at 16 23 27

- [x] Mobile Version
Screenshot 2024-07-12 at 16 21 43
Screenshot 2024-07-12 at 16 21 56

### PR DESCRIPTION
**Pull request by omprakashgupta1995**
Fix #3 

Test URLs:
- Before: https://main--piramal--aemsites.hlx.page
- After: https://issue-3--piramal--aemsites.hlx.page

### CODE REVIEW

**Review by asthabh23 — CHANGES_REQUESTED** [review]
Few observations on alignment:
1.  on reducing the browser width almost near `900px` , the content alignment needs to be made better.
Please introduce gaps within column so content is clearly visible for each cell.
Screenshot 2024-08-01 at 11 24 33

 Original site:
 
Screenshot 2024-08-01 at 11 25 33

 2. The social media icons need not be wrapped to be moved to next line. (also space between play store icons and right side content needs a recheck)
 
Screenshot 2024-08-01 at 11 27 39

3. we have added `clickable-images` class in columns block for social media icons - so that logo image followed by link should be wrapped within the link (this needs to be handled in DOM manipulation via JS).Please update so the icons are clickable.

File: blocks/footer/footer.css [hunk 1]
```diff
@@ -7,8 +7,437 @@ footer {
...
+  }
+
+  footer>.footer.block>div>.section:nth-child(4) .default-content-wrapper:nth-child(1) p:nth-child(1) {
+    margin-bottom: unset;
+  }
+
+  footer .footer.block .section.columns-container .columns-wrapper .columns.block div div p,
+  footer .footer.block .section.columns-container .columns-wrapper .columns.block div div p a {
+    color: #fff;
+    font-family: Nunito-Regular, sans-serif;
```
[the remark this answers is not in the document]
Comment (Sk-Zahid) [reply blocks/footer/footer.css:385]: Variables for font size 16 are not provided for body in styles.css also font weight

File: blocks/columns/columns.js [hunk 2]
```diff
@@ -13,6 +13,12 @@ export default function decorate(block) {
           picWrapper.classList.add('columns-img-col');
         }
       }
+      if (block.classList.contains('clickable-images')) {
```
Comment (mind-adobe) [inline blocks/columns/columns.js:16]: This should be handled in css by making the anchor tag width 100% height 100% and position absolute and displayed over the image. There shouldn't be an event listener for the click event.

**Comment by mind-adobe** [comment]
There is also a problem regarding columns spacing on some resolutions, please check it out.
image
image

**Review by asthabh23 — APPROVED** [review]
approving, Adobe team to cover the remaining parts

**Pull request merged**